Cargo.lock

```diff
@@ -12,24 +12,24 @@ dependencies = [
 
 [[package]]
 name = "addr2line"
-version = "0.13.0"
+version = "0.12.2"
```

Comment:
It looks like you ran `cargo update` here. Would it be possible to undo this?

Reply:
Running `git checkout origin/master -- Cargo.lock` and `cargo check` briefly would do nicely :relaxed: